packages/shared/src/components/tooltips/ChangelogTooltip.tsx

```diff
@@ -0,0 +1,118 @@
+import { postDateFormat } from '../../lib/dateFormat';
+import { Image } from '../image/Image';
+
+interface ChangelogTooltipProps<TRef> {
+  visible: boolean;
+  elementRef: MutableRefObject<TRef>;
+  onRequestClose?: (e?: React.MouseEvent | React.KeyboardEvent) => void;
+}
+
+function ChangelogTooltip<TRef extends HTMLElement>({
+  visible,
+  elementRef,
+  onRequestClose,
+}: ChangelogTooltipProps<TRef>): ReactElement {
+  // TODO WT-1054-changelog test extension
+  const isExtension = true; // !!process.env.TARGET_BROWSER;
+  const { user } = useContext(AuthContext);
+  const { data: post } = useQuery<FeedData, unknown, Post>(
+    [
+      'changelog',
+      'latest-post',
+      {
+        source: 'daily_updates',
+        first: 1,
+        loggedIn: !!user?.id,
+        unreadOnly: false,
+      },
+    ],
+    async ({ queryKey }) => {
+      const [, , variables] = queryKey;
+
+      return await request(graphqlUrl, SOURCE_FEED_QUERY, variables);
+    },
+    {
+      select: useCallback((data) => {
+        return data?.page.edges[0]?.node;
+      }, []),
+    },
+  );
+
+  return (
+    <BaseTooltip
+      content={
+        !!post && (
+          <div className="flex flex-col w-96 rounded-16 border shadow-2 focus:outline-none changelog bg-theme-bg-tertiary border-theme-color-cabbage">
+            <header className="flex flex-1 items-center py-3 px-4 border-b border-theme-divider-tertiary">
+              <Button className="bg-water-40 btn-primary small text-theme-label-secondary">
+                New release
+              </Button>
+              {onRequestClose && <ModalClose onClick={onRequestClose} />}
+            </header>
+            <section className="flex flex-col flex-1 p-5 h-full shrink max-h-full">
+              <Image
+                className="object-cover w-44 h-28 rounded-lg"
+                alt="Post Cover image"
+                src={post.image}
+                fallbackSrc={cloudinary.post.imageCoverPlaceholder}
+                loading="lazy"
+              />
+              <h3 className="mt-2 font-bold text-theme-label-primary typo-title3">
+                {post.title}
+              </h3>
+              <time
+                className="text-theme-label-quaternary typo-callout"
+                dateTime={post.createdAt}
+              >
+                {postDateFormat(post.createdAt)}
+              </time>
+              <div className="mt-2 w text-theme-label-tertiary typo-callout">
+                {post.summary}
+              </div>
+            </section>
+            <footer className="flex gap-3 items-center py-3 px-4 w-full h-16 border-t border-theme-divider-tertiary">
+              <Button className="btn-tertiary">Release notes</Button>
+              {isExtension && (
+                <Button className="bg-cabbage-40 btn-primary">
+                  Update extension
+                </Button>
+              )}
+            </footer>
+          </div>
```

Comment:
This is something we would most likely be using on the modal version as well. I think we can take this out and name a new component for this.

Reply:
What modal version?

Reply:
From the Sketch link, it looks like there will be a modal equivalent of this for the experiment.

![Screenshot 2023-02-13 at 8 04 31 PM](https://user-images.githubusercontent.com/13744167/218465905-ac4a20d6-4fe0-4738-a423-c4d25164b33a.png)

Reply:
From what I know the modal was the initial design, but one with a tooltip is the one we are going for. 

[Task page](https://dailydotdev.atlassian.net/wiki/spaces/HAN/pages/443351051/Change+log+popup#Parameters-for-A%2FB) does not mention any ab experiments, also the [referenced design](  https://www.sketch.com/s/2867ec63-0966-4647-b1d7-b6fe696ba251/p/9895C739-4D4B-4E24-BB62-E34B62CF6086/canvas) on ticket in Jira only shows the tooltip variant.

Let me know if I am missing something 🙏

Reply:
Oh, cool. I never knew about the path taken, and as you mentioned, there wasn't anything mentioned in the task about the AB experiment. Let me check with the team as it is a bit confusing.

Reply:
After checking with Seb there is no a/b test. Tnx @sshanzel for checking